src/transformers/models/auto/modeling_auto.py

```diff
@@ -434,6 +435,7 @@
 MODEL_FOR_INSTANCE_SEGMENTATION_MAPPING_NAMES = OrderedDict(
     [
         # Model for Instance Segmentation mapping
+        ("mask2former", "Mask2FormerForUniversalSegmentation"),
```

Comment:
You can add Mask2Former to the new "universal segmentation" mapping which will be added in #20766

Reply:
Sure, I will add it after your PR is merged